frontend/src/modules/Shares/index.js

```diff
@@ -1 +1 @@
-export { ShareInboxList } from './components';
+export { ShareBoxList } from './components';
```

Comment:
Merge conflict here - need to pull latest from main to check to load `SharesModule` the way we are now managing frontend dependencies

May resolve the "Manage Dependency TODO" earlier